scripts/validate_rst_title_capitalization.py

```diff
@@ -233,6 +232,8 @@ def main(source_paths: List[str], output_format: str) -> bool:
 
     for filename in find_rst_files(source_paths):
         for title, line_number in find_titles(filename):
+            if ":class" in title:
+                print(title)
```

Comment:
I *think* this should be:

```python
if ":class" in title:
   continue
```

Reply:
Horrible... I forgot to remove it. Correcting it right away!

Reply:
Hmm Im not so much used to git yet.. Apart from that, I didn't roll back previous changes that I made, I thought it will be ok. I don't know if the changes i didn't remove will fail

Reply:
Or if it conflicts with previous PR or something